src/main/kotlin/app/utils/PathValidator.kt

```diff
@@ -8,11 +10,11 @@ object PathValidator : IParameterValidator {
     override fun validate(name: String, value: String) {
         if (!isValidPath(value)) {
             throw ParameterException("Parameter $name should be "
-                    + "correct email (found $value)")
+                    + "correct path (found $value)")
```

Comment:
It will probably be more readable if you break the line right after ParameterException:

```
throw ParameterException(
    "Parameter $name should be a correct path (found $value)")
```